openage/convert/service/init/api_export_required.py

```diff
@@ -16,7 +16,7 @@
     from openage.util.fslike.union import UnionPath
 
 
-CURRENT_API_VERSION = "0.4.0"
+CURRENT_API_VERSION = "0.5.1"
```

Comment:
we should import the global version id here, no?

Reply:
This was more of a test because I thought the API modpack might have been wrongly generated and caused the error in https://github.com/SFTtech/openage/issues/1569 . But this was probably not the issue.

I don't think the modpack version should be bumped with every engine release btw. but maybe if there's a new version of the engine modpack, it should have the same version no as the current engine version.

Reply:
i might close this PR and think of something else to fix the issue